output_operation.go

```diff
@@ -65,21 +60,19 @@ func gceUnkownAggregatedList(ctx context.Context, computeService *compute.Servic
 	req = req.Filter(os.Getenv("OPERATION_FILTER"))
 
 	if err := req.Pages(ctx, func(page *compute.OperationAggregatedList) error {
-		var c = make(chan bool, len(page.Items))
+		var c = make(chan map[string][]*OutputOperation, len(page.Items))
 
 		for _, operationsScopedList := range page.Items {
-			go func(operationsScopedList compute.OperationsScopedList) {
-				lenOperationsScopedListOperations := len(operationsScopedList.Operations)
-
-				if lenOperationsScopedListOperations > 0 {
-					unkownAggregatedList = multiCheckOperation(ctx, lenOperationsScopedListOperations, operationsScopedList)
```

Comment:
issue is here.
Because the page.Item will return same project but different zone, the next zone will overwrite the exist one.